_includes/comment-form.html

```diff
@@ -1,46 +1,57 @@
         <div id="post-comments" class="post-comments"></div>
+
         <!-- New comment form -->
-        <form id="comment-form">
+        <form id="comment-form" class="comment-form">
           <h4>Leave a comment</h4>
-          <div class="comment-name">
-            <p>Name</p> <input type='text' name='name' id='name' required /><br>
+
+          <div class="comment-form__field comment-form__field--name">
+            <label class="comment-form__field__label" for="name">Name</label>
+            <input type="text" name="name" id="name" required>
           </div>
 
-          <div class="comment-email">
-            <p>Email</p> <input type='email' name='email' id='email' required />
-            <p class="email-disclaimer">Savas Labs will never sell your email address or spam you.</p><br>
+          <div class="comment-form__field comment-form__field--email">
+            <label class="comment-form__field__label" for="email">Email</label>
+            <input type="email" name="email" id="email" required>
+            <p class="comment-form__field__helptext">Savas Labs will never sell your email address or spam you.</p>
           </div>
 
-          <p>Comment</p>
-          <textarea name='comment' id='comment' required></textarea>
-          <p class="hint below-textarea">Plain text format only please.</p>
+          <div class="comment-form__field comment-form__field--comment">
+            <label class="comment-form__field__label" for="comment">Comment</label>
+            <textarea name="comment" id="comment" required></textarea>
+            <p class="comment-form__field__helptext">Plain text format only please.</p>
+          </div>
 
-          <input type='hidden' name='slug' id='slug' value='{{ page.url }}' />
-          <input type='text' name='url' id='url' />
+          <input type="hidden" name="slug" id="slug" value="{{ page.url }}">
+          <input type="text" name="url" id="url">
 
-          <div class='are-you-human'>
-            <div class='are-you-human-owl'>
+          <div class="comment-form__field comment-form__field--nocaptcha">
+            <div class="comment-form__field--nocaptcha__image">
               <img src="/assets/img/icons-and-logos/owl.png" alt="Savas Labs logo">
             </div>
-            <div class='are-you-human-field'>
-              <p>What type of animal is the Savas Labs logo?</p>
-              <input type='text' name='nocaptcha' id='nocaptcha' class='nocaptcha'>
-              <p class="hint">Hint: 3 letters long, starts with an 'o' and ends with an 'l'.</p>
+            <div class="comment-form__field--nocaptcha__input">
+              <label class="comment-form__field__label" for="nocaptcha">What type of animal is the Savas Labs logo?</label>
+              <p class="comment-form__field__helptext">Hint: 3 letters long, starts with an "o" and ends with an "l".</p>
+              <input type="text" name="nocaptcha" id="nocaptcha">
```

Comment:
@kostajh I went ahead and added it. I intend to remove this field but want proper markup in the file's history in case we ever decide to restore it. Thanks for catching!